utils/db_setup/studentIsInSection.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE "studentIsInSection"(
+       id        serial,
+       start     serial,
```

Comment:
serials are autoincrement, these should just be int or a real date type

Reply:
If you do not use the default value, it acts as a normal unsigned integer type. I believe you can directly insert NULL. Postgres is wacky. Could be wrong here though but that's what I understood from reading the docs.

Reply:
[Postgres Numeric Datatypes](https://www.postgresql.org/docs/11/datatype-numeric.html)

Actually I probably should change it either way because

`integer` - `4 bytes` - `-2147483648 to +2147483647`
`serial` - `4 bytes` - `1 to 2147483647`

I am not sure if the documentations was incorrect due to some copy-paste magic, but it doesn't make sense for the same number of bytes for the positive range to be the same.